kyaml/resid/gvk.go

```diff
@@ -210,14 +216,16 @@ func (x Gvk) IsLessThan(o Gvk) bool {
 // anything in the corresponding fields, e.g.
 //
 // this item:
-//       <Group: "extensions", Version: "v1beta1", Kind: "Deployment">
+//
+//	<Group: "extensions", Version: "v1beta1", Kind: "Deployment">
 //
 // is selected by
-//       <Group: "",           Version: "",        Kind: "Deployment">
+//
+//	<Group: "",           Version: "",        Kind: "Deployment">
 //
 // but rejected by
-//       <Group: "apps",       Version: "",        Kind: "Deployment">
 //
+//	<Group: "apps",       Version: "",        Kind: "Deployment">
```

Comment:
Hi. I'm just passing by. Is this change due to IDE formatting?

Reply:
Hmm looks like it. I'll put it back! 🙂

Reply:
OK, I realized this diff was actually caused by my IDE running the gofmt in the background. I believe it would be better if we just stick to the current formatted version of the comment.